client/components/TestQueue/index.jsx

```diff
@@ -141,26 +143,33 @@ class TestQueue extends Component {
 TestQueue.propTypes = {
     cycle: PropTypes.object,
     cycleId: PropTypes.number,
-    testsForRuns: PropTypes.object,
+    testsByRunId: PropTypes.object,
     usersById: PropTypes.object,
     userId: PropTypes.number,
+    testsFetched: PropTypes.bool,
     dispatch: PropTypes.func
 };
 
 const mapStateToProps = (state, ownProps) => {
-    const { cyclesById, runsForCycle } = state.cycles;
+    const { cyclesById, testsByRunId } = state.cycles;
     const { usersById } = state.users;
     const userId = state.user.id;
     const cycleId = parseInt(ownProps.match.params.cycleId);
 
     let cycle = cyclesById[cycleId];
-
-    let testsForRuns = undefined;
-    if (runsForCycle && runsForCycle[cycleId]) {
-        testsForRuns = runsForCycle[cycleId];
+    let testsFetched = true;
```

Comment:
thanks :) Re: starting with true -- I just wanted to set to false the first time I found missing data, rather than set to true after confirming all data was there!